PhpAmqpLib/Wire/IO/SocketIO.php

```diff
@@ -314,6 +314,15 @@ protected function setErrorHandler(): void
         socket_clear_error($this->sock);
     }
 
+    private function selectProtocol() :int
+    {
+        if (is_null($this->config) || is_null($this->config->getSocketProtocolMode())) {
```

Comment:
can we improve readability here?
```
if ($this->config && $this->config->getSocketProtocol()){
return $this->config->getSocketProtocol();
}
```

Reply:
Why not using a simple flag forcing the AF_INET instead of selecting the version by hand ?

It will probably be easier to understand. What do you think ?